.gitignore

```diff
@@ -3,6 +3,8 @@ node_modules/
 
 # build directory
 dist/
+icons/
+!extension/src/static/icons/
```

Comment:
Do we want to ignore icons?

Reply:
No need to ignore icons. I had uncommitted local `icons/` right under `extension/` for some reason..
I'm removing these two lines